pkgs/tools/graphics/wkhtmltopdf-bin/default.nix

```diff
@@ -1,28 +1,81 @@
-{ stdenv, lib, fetchurl, wkhtmltopdf, xar, cpio }:
+{ autoPatchelfHook
+, cpio
+, freetype
+, zlib
+, openssl
+, dpkg
+, fetchurl
+, gcc-unwrapped
+, lib
+, libjpeg8
+, libpng
+, fontconfig
+, stdenv
+, wkhtmltopdf
+, xar
+, xorg
+}:
 
-stdenv.mkDerivation rec {
+let
+  inherit (lib)
+    getLib
+    optionalAttrs
```

Comment:
Worth it to replace two `lib.`s? This is a bit annoying for static analysis of nix files.